frontend/src/app/features/leadership-experiences/leadership-experiences-type/leadership-experiences-type.service.spec.ts

```diff
@@ -0,0 +1,55 @@
+import { TestBed } from '@angular/core/testing';
+
+import { LeadershipExperiencesTypeService } from './leadership-experiences-type.service';
+import { provideHttpClient } from '@angular/common/http';
+import { HttpTestingController, provideHttpClientTesting } from '@angular/common/http/testing';
+import { LeadershipExperiencesTypeModel } from './leadership-experiences-type.model';
+import {
+  leadershipExperienceType1,
+  leadershipExperienceType2
+} from '../../../shared/test/test-data';
+
+describe('LeadershipExperiencesTypeService', () => {
+  let service: LeadershipExperiencesTypeService;
+  let httpMock: HttpTestingController;
+  const API_URL = '/api/v1/leadership-experiences-types';
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [LeadershipExperiencesTypeService,
+        provideHttpClient(),
+        provideHttpClientTesting()]
+    })
+      .compileComponents();
+
+    service = TestBed.inject(LeadershipExperiencesTypeService);
+    httpMock = TestBed.inject(HttpTestingController);
+  });
+
+  afterEach(() => {
+    httpMock.verify();
+  });
+
+  it('should be created', () => {
+    expect(service)
+      .toBeTruthy();
+  });
+
+  describe('LeadershipExperiencesType CRUD endpoints', () => {
+    it('should call httpClient.get with the correct URL and return the experience with the correct id', () => {
```

Comment:
If you already copy half of the test make sure to at least change the model name of it . ☝️  
Also your test does not rely on a id but rather calls the get-all endpoint.

```suggestion
    it('should call httpClient.get with the correct URL and return all the leadership-experience-types', () => {
```